boards/arduino-mega2560/include/periph_conf.h

```diff
@@ -264,23 +265,110 @@ extern "C" {
 /**
  * @brief GPIO configuration
  */
-#define GPIO_NUMOF          (0U)
-#define GPIO_0_EN           0
-#define GPIO_1_EN           0
-#define GPIO_2_EN           0
-#define GPIO_3_EN           0
-#define GPIO_4_EN           0
-#define GPIO_5_EN           0
-#define GPIO_6_EN           0
-#define GPIO_7_EN           0
-#define GPIO_8_EN           0
-#define GPIO_9_EN           0
-#define GPIO_10_EN          0
-#define GPIO_11_EN          0
-#define GPIO_12_EN          0
-#define GPIO_13_EN          0
-#define GPIO_14_EN          0
-#define GPIO_15_EN          0
+#define GPIO_NUMOF          (14U)
+#define GPIO_0_EN           1 /* but not used realy */
+#define GPIO_1_EN           1 /* but not used realy */
+#define GPIO_2_EN           1
+#define GPIO_3_EN           1
+#define GPIO_4_EN           1
+#define GPIO_5_EN           1
+#define GPIO_6_EN           1
+#define GPIO_7_EN           1
+#define GPIO_8_EN           1
+#define GPIO_9_EN           1
+#define GPIO_10_EN          1
+#define GPIO_11_EN          1
+#define GPIO_12_EN          1
+#define GPIO_13_EN          1
+
+typedef struct {
+  uint8_t port;
+} GPIO_TypeDef;
+
+
+/* GPIO helper makros */
+#define GPIO_PORT_B         PORTB
+#define GPIO_PORT_E         PORTE
+#define GPIO_PORT_G         PORTG
+#define GPIO_PORT_H         PORTH
+#define GPIO_DDR_B          DDRB
+#define GPIO_DDR_E          DDRE
+#define GPIO_DDR_G          DDRG
+#define GPIO_DDR_H          DDRH
+#define GPIO_PIN_B          PINB
+#define GPIO_PIN_E          PINE
+#define GPIO_PIN_G          PING
+#define GPIO_PIN_H          PINH
```

Comment:
Why is this needed? Well it doesn't hurn so much but there is no reason for it, right? In my opinion it would be totally fine to use `#define GPIO_2_PORT         PORTE` in the lower lines